src/Rpc.proto

```diff
@@ -184,6 +184,18 @@ message GetPeerReputationResponse {
    int32 reputation = 1;
 }
 
+message GetPeerBlackListRequest {
```

Comment:
As discussed, this name is a bit more accurate if you want to change the peer's status
```suggestion
message UpdatePeerBlackListRequest {
```